Assets/Scripts/Editor.cs

```diff
@@ -241,8 +247,22 @@ void Update ()
 
 		if (editorState == EditorState.Active)
 		{
-			if (Input.GetMouseButtonDown(0) 
-				&& !EventSystem.current.IsPointerOverGameObject() 
+			var ignoreRaycast = false;
```

Comment:
Why this block?

Reply:
To prevent the raycast from creating a new interaction point while pressing play.